app/views/user/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>List of all User from users list</h1>
```

Comment:
- [ ] This change is for the folder containing this file, not the file itself. The folder should be `users` to match the controller, not `user`.